docs/delta_backups.md

```diff
@@ -60,3 +60,13 @@ Then, only the diff will be sent.
 1. edit the job to target the other remote
 1. copy files from the first remote to the other one
 1. run the backup (delta)
+
+## Full backup interval
+
+This advanced setting defines the number of backups after which a full backup is triggered, ie the maximum length of a delta chain.
+
+For example, with a value of 2, the first two backups will be a full and a delta, and the third will start a new chain with a full backup.
```

Comment:
Which value?

Reply:
It's in the full backup interval context.

Reply:
Also, @Fohdeesha already reviewed and corrected the doc, I believe it would have changed it if it was unclear :slightly_smiling_face: